frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleashTooltip.tsx

```diff
@@ -0,0 +1,127 @@
+import type * as React from 'react';
+import type { FC, ReactNode } from 'react';
+import { HtmlTooltip } from 'component/common/HtmlTooltip/HtmlTooltip';
+import {
+    Box,
+    Button,
+    Link,
+    styled,
+    Typography,
+    ClickAwayListener,
+} from '@mui/material';
+import { type Link as RouterLink, useNavigate } from 'react-router-dom';
+import OpenInNew from '@mui/icons-material/OpenInNew';
+import { ReactComponent as UnleashLogo } from 'assets/img/logoWithWhiteText.svg';
+
+const Header = styled(Box)(({ theme }) => ({
+    backgroundColor: theme.palette.primary.light,
+    color: theme.palette.primary.contrastText,
+    borderRadius: theme.spacing(1, 1, 0, 0), // has to match the parent tooltip container
```

Comment:
There are items on theme for border-radius. Check `theme.shape....` - it might require you to use template literal with px (`${theme...}px`)

Reply:
But maybe you can have "overflow:hidden" on parent?